paimon-spark/paimon-spark-common/src/test/scala/org/apache/paimon/spark/PaimonTableTest.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.spark
+
+import org.apache.spark.sql.test.SharedSparkSession
+
+import scala.collection.mutable
+
+trait PaimonTableTest extends SharedSparkSession {
+
+  val bucket: Map[String, String]
+
+  def appendPrimaryKey(primaryKeys: Seq[String], props: mutable.Map[String, String]): Unit
+
+  def createTable(
+      tableName: String,
+      columns: String,
+      primaryKeys: Seq[String],
+      partitionKeys: Seq[String] = Seq.empty,
+      props: Map[String, String] = Map.empty): Unit = {
+    val newProps: mutable.Map[String, String] = mutable.Map.empty[String, String] ++ bucket ++ props
```

Comment:
What if props contains `bucket`? Should it be banned.

Reply:
then should respect the props's bucket.